apps/sms/test/marionette/drafts_test.js

```diff
@@ -124,19 +109,18 @@ marionette('Messages Drafts', function() {
   });
 
   suite('Messages Drafts Test Suite', function() {
-    var smsMessage, draft, messagesStorage;
+    var conversation, draft, inboxView;
+
+    function relaunchApp() {
+      messagesApp.close();
+      messagesApp.launch();
+      storage.setMessagesStorage(
```

Comment:
I think in the future we can retrieve current storage from app before closing, so that we can relaunch it with the same storage state  (e.g. without messages that were deleted). Absence of this feature didn't prevent me from covering uses cases I had, so I didn't do it :)